pkg/apis/logging/v1/elasticsearch_types.go

```diff
@@ -58,12 +58,23 @@ type ElasticsearchStatus struct {
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
 
 	Nodes                  []ElasticsearchNodeStatus             `json:"nodes"`
-	ClusterHealth          string                                `json:"clusterHealth"`
+	ClusterHealth          ClusterHealth                         `json:"clusterHealth"`
```

Comment:
@jcantrill we can't change this since this is going out after 4.1 can we...
I'm considering just making this new field called `Cluster`

Reply:
Probably Not.